exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/store/LocalPersistentStore.java

```diff
@@ -61,16 +63,29 @@
   private final AutoCloseableLock readLock = new AutoCloseableLock(readWriteLock.readLock());
   private final AutoCloseableLock writeLock = new AutoCloseableLock(readWriteLock.writeLock());
 
+  //Provides a threshold above which we report the time to load
+  private static final long LISTTIME_THRESHOLD_MSEC = 2000L;
```

Comment:
`LISTTIME_THRESHOLD_MSEC` -> `LIST_TIME_THRESHOLD_MSEC`

Reply:
OK. Will fix this.